tests/middleware/test_proxy_headers.py

```diff
@@ -65,7 +65,7 @@ def make_httpx_client(
     "::11.22.33.44",  # This is a dual address
 ]
 _TRUSTED_IPv6_NETWORKS = "2001:db8:abcd:0012::0/64"
-_TRUSTED_LITERALS = "some-literal , unix:///foo/bar  ,  /foo/bar"
+_TRUSTED_LITERALS = "some-literal , unix:///foo/bar  ,  /foo/bar, garba*gewith*"
```

Comment:
We probably have a test case with `*`, can you add `[*]` as well?

Reply:
Added test case with trust host using `["*"]`, It's similar to the existing trust everything so I did not add every address as an input.